site/themes/arangodb-docs-theme/static/js/codeblocks.js

```diff
@@ -1,6 +1,6 @@
 
 function switchTab(tabGroup, event) {
-    var tabId = event.target.value;
+    var tabId = event.target.innerHTML;
```

Comment:
Potentially safer?
```suggestion
    var tabId = event.target.textContent;
```

Reply:
The logic there has changed, instead of the event i just need the tab name i'm going to switch to. Check latest commit